warehouse/bigquery.go

```diff
@@ -37,11 +37,35 @@ func NewBigQuery(c *config.Config) *BigQuery {
 	if c.GCS.GCSOnly {
 		log.Printf("Config flag GCSOnly is on, data will not be loaded to BigQuery")
 	}
+
 	return &BigQuery{
 		conf: c,
 	}
 }
 
+// GetExportTableColumns returns a slice of the columns in the existing export table
+func (bq *BigQuery) GetExportTableColumns() []string {
+	var columns []string
```

Comment:
NIT: maybe move this to line 62? creating variables closer to where it is used is slightly easier to grok.

Reply:
+1